pkg/instrumentation/defaultinstrumentation_test.go

```diff
@@ -68,6 +69,23 @@ func Test_getDefaultInstrumentation(t *testing.T) {
 					{Name: "OTEL_LOGS_EXPORTER", Value: "none"},
 				},
 			},
+			DotNet: v1alpha1.DotNet{
```

Comment:
@vastin can you confirm these env variables?

Reply:
It is correct.